### PR TITLE
feat: add bribe processor

### PR DESCRIPTION
**Pull request by mitche50**

### CODE REVIEW

File: src/vaults/src/handlers/bribe-processor-handler.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { TreeDistribution } from '../../generated/BribeProcessor/VotiumBribeProcessor';
+import { BadgerTreeDistribution } from '../../generated/schema';
+import { BVECVX } from '../constants';
+
+export function handleTreeDistribution(event: TreeDistribution): void {
```
Comment (axejintao) [inline src/vaults/src/handlers/bribe-processor-handler.ts:5]: https://github.com/Badger-Finance/badger-subgraph/blob/e375b7fc37486ca6b4cc75178db52b79e2f8d667/src/vaults/src/handlers/strategy-handler.ts#L78-L85

we could use this one instead

File: src/vaults/src/handlers/bribe-processor-handler.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  let entity = new BadgerTreeDistribution(event.transaction.from.toHex());
+
+  entity.id = event.transaction.hash.toHex() + '-' + event.logIndex.toString();
```
Comment (axejintao) [inline src/vaults/src/handlers/bribe-processor-handler.ts:8]: `.toHexString()`

File: src/vaults/src/handlers/bribe-processor-handler.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  entity.token = event.params.token.toHex();
```
Comment (axejintao) [inline src/vaults/src/handlers/bribe-processor-handler.ts:9]: `loadToken(t.toHexString()).id` so in the case this was the first time the graph ever saw this token it gets saved

**Comment by mitche50** [comment]
test deployment here: https://thegraph.com/hosted-service/subgraph/mitche50/badger-vaults

**Review by axejintao — APPROVED** [review]

**Pull request merged**